cosmo_tester/test_suites/cluster/create_replace_certs_config_script.py

```diff
@@ -0,0 +1,49 @@
+import argparse
```

Comment:
This should probably be somewhere in the resources tree (cosmo_tester/resources/<something>)? 
It doesn't look like it's actually directly invoked test code, but instead a supporting script that is used elsewhere?

Reply:
Correct, moved to be under resources/scripts.